core/src/test/java/org/apache/iceberg/rest/RESTCatalogServlet.java

```diff
@@ -53,7 +54,7 @@ public class RESTCatalogServlet extends HttpServlet {
   private static final Logger LOG = LoggerFactory.getLogger(RESTCatalogServlet.class);
 
   private final RESTCatalogAdapter restCatalogAdapter;
-  private final Map<String, String> responseHeaders =
+  private final Map<String, String> defaultResponseHeaders =
```

Comment:
Why is this not a private static final constant?

Reply:
I just renamed it, but thanks for the suggestion, I changed it accordingly